src/content/app/tools/vep/views/vep-submissions/listed-vep-submission/ListedVepSubmission.module.css

```diff
@@ -13,8 +13,7 @@
   border: 1px solid var(--color-grey);
   padding-left: var(--standard-gutter);
   padding-right: var(--standard-gutter);
-  padding-top: 24px;
-  padding-bottom: 24px;
+  line-height: 60px;
```

Comment:
Spinner off-center now.

<img width="1035" alt="image" src="https://github.com/user-attachments/assets/8b24695e-10a6-40cd-ad82-485eebe0e78a">

But line-height is not the right approach anyway. It will lead to this:

![image](https://github.com/user-attachments/assets/bdd7b85e-89c0-4c02-80bd-4f4b73c5d0b5)